daemon/cmd/status.go

```diff
@@ -192,6 +192,21 @@ func (d *Daemon) getHostFirewallStatus() *models.HostFirewall {
 	}
 }
 
+func (d *Daemon) getSRv6Status() *models.SRv6 {
+	mode := models.SRv6ModeDisabled
+	if option.Config.EnableSRv6 {
+		mode = models.SRv6ModeEnabled
+	}
+	devices := make([]string, len(option.Config.Devices))
+	for i, iface := range option.Config.Devices {
+		devices[i] = iface
+	}
```

Comment:
The build currently fails because of this code. See how it is done for the host firewall above. Note that the codebase evolved since the commit that added the similar logic for the host firewall.